bin/taxref_reformat_unite.sh

```diff
@@ -7,7 +7,7 @@
 tar xzf *.gz
 
 # Remove leading "k__" and the like and replace space with underscore to create assignTaxonomy.fna
-cat */*.fasta | sed 's/;s__.*//' | sed 's/[a-z]__//g' | sed 's/ /_/g' > assignTaxonomy.fna
+cat */developer/*.fasta | sed 's/;[ks]__.*//' | sed 's/[a-z]__//g' | sed 's/ /_/g' > assignTaxonomy.fna
```

Comment:
If I understand this correctly, spaces in the actual sequence (not only header) are replaced by underscore. I suggest to remove the spaces in the sequnces instead of replacing them.

Reply:
> If I understand this correctly, spaces in the actual sequence (not only header) are replaced by underscore. I suggest to remove the spaces in the sequnces instead of replacing them.

It was meant to be in the names -- easier to get the spaces back with '_' -- and I should change the script accordingly.